### ISSUE TITLE
Picard 2.10 released, Update docker-picard to 2.10?

### ISSUE THREAD

**Issue by louhisuo** [mikenye/docker-picard#76]
Any plans to upgrade docker image to Pickard 2.10, now when it is released?
https://blog.metabrainz.org/2023/10/12/picard-2-10-released/

**Comment by mia-riezebos** [comment]
would love this

**Comment by FinalDoom** [comment]
PR #75 claims to address this. Would love to get @mikenye to look at that

**Comment by phw** [comment]
@mikenye I tried the new image from dockerhub. There seems to be an issue with the SSL root certificates, despite the call to `update-ca-certificates` in https://github.com/mikenye/docker-picard/blob/master/Dockerfile#L107

**Comment by mikenye** [comment]
@phw can you provice more information on the issue?

**Comment by xxxarmitagexx** [comment]
Basically all the calls to musicbrainz.org fail:

[app         ] E: 23:43:45,093 webservice.ssl_errors:363: SSL error: The peer did not present any certificate

[app         ] E: 23:43:45,093 webservice._handle_reply:534: Network request error for https://musicbrainz.org/ws/2/release/5f0922e6-c2a3-4644-8eb8-f383347033dd?inc=aliases%2Bannotation%2Bartist-credits%2Bartist-rels%2Bartists%2Bcollections%2Bdiscids%2Bisrcs%2Blabels%2Bmedia%2Brecording-rels%2Brecordings%2Brelease-group-level-rels%2Brelease-groups%2Brelease-rels%2Bseries-rels%2Burl-rels%2Bwork-rels -> SSL handshake failed (QT code 6, HTTP code 0)

[app         ] E: 23:43:45,093 ui/item.error_append:108: <Album 5f0922e6-c2a3-4644-8eb8-f383347033dd ''>: SSL handshake failed

**Comment by mikenye** [comment]
Ok thanks, I'm checking it out now.

**Comment by mikenye** [comment]
Should be fixed now. Please check and let me know.

**Comment by xxxarmitagexx** [comment]
Thanks it works normally now.

**Comment by louhisuo** [comment]
[musicbrainz-picard_musicbrainz-picard-custom-app-6947b487-tzsn9_musicbrainz-picard-custom-app.log](https://github.com/mikenye/docker-picard/files/13265748/musicbrainz-picard_musicbrainz-picard-custom-app-6947b487-tzsn9_musicbrainz-picard-custom-app.log)
For me 2.10 version do not launch (see attached logs), however 2.8.5 works normally.

**Comment by mikenye** [comment]
@louhisuo can you please try running with no mapped volumes, and see if the application starts?

**Comment by louhisuo** [comment]
No, it did not start even I removed both volume definitions. Will next try to redeploy from the scratch.

**Comment by louhisuo** [comment]
Got it running by adding following Linux capabilities (KILL and NET_BIND_SERVICE) to my Kubernetes deployment YAML due to implementation of Helm chart used to deploy Picard. These capabilities coming from baseimage from jlesage and  changes will not be needed general use (specific for Kubernetes implementation I am using).

However, you need to update your baseimage because CJK fonts are broken without. Take a look
[docker-baseimage-gui  v4.5.1](https://github.com/jlesage/docker-baseimage-gui/releases/tag/v4.5.1)
